### PR TITLE
[CIVP-12081] BUG Allow non-seekable buffer inputs to file_to_civis

### PR DESCRIPTION
**Pull request by jkulzick**
… for a seekable file object

Included an additional check beyond HAS_TOOLBELT in civis.io.file_to_civis to ensure the file object is streamable (buf.seekable()). This should enable zip file objects to be uploaded using file_to_civis even when the requests_toolbelt is installed.

### CODE REVIEW

File: civis/tests/test_io.py [hunk 1]
```diff
@@ -99,6 +100,27 @@ def test_get_url_from_file_id(self, *mocks):
...
+                with zip_file.open(zip_member) as zip_member_buf:
+                    result = civis.io.file_to_civis(zip_member_buf, zip_member)
+
+        assert isinstance(result, int)
+
+    @mock.patch(api_import_str, return_value=civis_api_spec)
+    def test_file_to_civis(self, *mocks):
+        with tempfile.NamedTemporaryFile() as tmp:
+            tmp.write(b'a,b,c\n1,2,3')
+            tmp.flush()
```
Comment (stephen-hoover) [inline civis/tests/test_io.py]: This would also need a `tmp.seek(0)` so that we read the entire buffer, yes?

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 - Fixed a bug where the version of a dependency for Python 2.7 usage was incorrectly specified.
 
+## 1.5.3 - 2017-05-18
+### Changed
+- Changed criteria in ``civis.io.file_to_civis`` for streaming a file to include a check on if the file object is seekable.
```
Comment (stephen-hoover) [inline CHANGELOG.md]: The thing that users care about is that they can now provide file-like objects which aren't seekable, right? We used to error on that. I'd rephrase this entry to make that clear.

**Comment by jkulzick** [comment]
Latest commit 6d792be:

1. Moved entry in CHANGELOG to the [unreleased] section and re-worded it
2. Updated comment block in civis.io.file_to_civis to also comment on new seekable criteria
3. removed change to version in _version
4. added tmp.seek(0) to test_file_to_civis prior to calling civis.io_file_to_civis

**Comment by beckermr** [comment]
We need to make JIRA tickets for this stuff in the future, @jkulzick

**Comment by jkulzick** [comment]
@beckermr @stephen-hoover 

Thanks for the jira examples @beckermr! I filed this jira. Hopefully it looks good
https://civisanalytics.atlassian.net/browse/CIVP-12081

**Comment by beckermr** [comment]
LGTM on the JIRA! Thanks!
